src/components/pages/Chat/Message.tsx

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components'
+
+type Props = {
+    className?: string,
+    body?: string,
+    createdAt?: number,
+    from: string,
+}
+
+const Body = styled.div`
+    background-color: #F1F4F7;
+    border-radius: 0.75rem;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    max-width: 100%;
+    padding: 0.625rem 1rem;
+`
+
+const AvatarWrap = styled.div`
+    flex-shrink: 0;
+    margin-right: 1rem;
+`
+
+const Avatar = styled.div`
+    background-color: #F1F4F7;
+    border-radius: 50%;
+    height: 2.5rem;
+    width: 2.5rem;
+`
+
+const ME = '0xa047314ff08Cead7726Ceb90fc0CB39D80607227'
```

Comment:
Not sure what this is, @bearni95 any ideas?

Reply:
It's "work in progress". ;)